crates/bevy_winit/src/lib.rs

```diff
@@ -287,10 +300,12 @@ impl Default for WinitAppRunnerState {
         Self {
             active: ActiveState::NotYetStarted,
             window_event_received: false,
+            device_event_received: false,
             redraw_requested: false,
             wait_elapsed: false,
             last_update: Instant::now(),
             scheduled_update: None,
+            start_forced_updates: 5,
```

Comment:
A comment here about how this number was chosen would help a lot.

Reply:
very scientifically!
on the windows machine I tested I was getting 1 event at start, on my mac it's 3, so 5 has a safe margin of error